packages/office-ui-fabric-react/src/components/DetailsList/ShimmeredDetailsList.types.ts

```diff
@@ -5,7 +5,7 @@ import { IStyleFunctionOrObject } from '../../Utilities';
 
 /**
  * ShimmeredDetailsList props interface
- * {@docCategory ShimmeredDetailsList}
+ * {@docCategory DetailsList}
```

Comment:
@natalieethell , @KevinTCoughlin I have changed the `docCategory` tag to `DetailsList`. I think it's more appropriate to have it on the DetailsList page than on Reference page.

Reply:
Seems reasonable, but even `DetailsList` is a bit of a catch-all at this point. Is there any way or usefulness to get these interfaces on https://developer.microsoft.com/en-us/fabric#/controls/web/detailslist/shimmer?

Reply:
I don't think we are placing any docs on subpages. @natalieethell can you confirm? Is there a way to easily add them to that page? Now it's either `DetailsList` or `References`. So where should we place it?

Reply:
`DetailsList` is fine if its not easy.

Reply:
Ah, yeah with the current implementation, it's not possible to place docs on subpages—although I could see the potential need for that. I think placing it under `DetailsList` makes sense to me!